tests/test_self_chat.py

```diff
@@ -29,3 +29,16 @@
             ]
         )
         self_chat.self_chat(opt)
+
+    def test_ed(self):
+        pp = self_chat.setup_args()
+        opt = pp.parse_args(
+            [
+                '-mf',
+                'zoo:blender/blender_3B/model',
```

Comment:
This is a very heavy model for a small test. Can we use the tutorial generator?

Reply:
Yeah you're right about that, will change it.

Reply:
Yeah you're right, I'll change it